package.json

```diff
@@ -69,14 +69,14 @@
     "trailingComma": "all"
   },
   "peerDependencies": {
-    "@adonisjs/ace": "^6.0.0",
-    "@adonisjs/core": "^5.0.0-preview-rc-1.12",
-    "@adonisjs/fold": "^6.0.0"
+    "@adonisjs/ace": "^7.1.5",
+    "@adonisjs/fold": "^7.0.7"
   },
   "devDependencies": {
-    "@adonisjs/ace": "^6.8.0",
-    "@adonisjs/fold": "^6.3.0",
-    "@adonisjs/logger": "^3.0.1",
+    "@adonisjs/core": "^5.0.2-beta-rc-7",
+    "@adonisjs/ace": "^7.1.5",
+    "@adonisjs/fold": "^7.0.7",
```

Comment:
is this still used somewhere? virk told me we should be able to remove this dependency